src/views/ProductInventory.vue

```diff
@@ -296,16 +298,39 @@ export default {
     IonThumbnail,
     IonTitle,
     IonToolbar,
+  },
+   computed: {
+    ...mapGetters({
+     pinnedPages: 'product/getPinnedPages'
+    })
   },
   setup () {
     return {
       downloadOutline,
       folderOutline,
       swapVerticalOutline,
       sync,
-      filterOutline
+      filterOutline,
     };
   },
+  data(){
+    return{
+      pagePinned: false,
+    }
+  },
+   methods:{
+    pinned(){
+      this.pagePinned = !this.pagePinned;
+      if(this.pagePinned){
+        this.pinnedPages.push("/product");
+        console.log(this.pinnedPages)
+      }
+      else{
+       this.pinnedPages = "";
```

Comment:
Why we are assigning empty string to pinnedPages property?

Reply:
Previously we were using an object instaed of an array, so to empty a slot I assigned empty string to pinned pages. Now I have updated the code to remove that element from array.